### PR TITLE
Feature/pelagos 5339 additions to go mri quarterly report

### PR DESCRIPTION
**Pull request by fightingtexasaggie**

### CODE REVIEW

File: src/Repository/LogActionItemRepository.php [hunk 1]
```diff
@@ -90,4 +90,51 @@ public function countDownloads(\DateTime $start = null, \DateTime $stop = null):
 
         return $downloadCount;
     }
+
+    /**
+     * Array downloads, optional date range and also filter aware.
+     *
+     * @return array the count of the datasets downloaded, per FAIR guidelines
+     */
+    public function getDownloads(): array
```
Comment (mickel1138) [inline src/Repository/LogActionItemRepository.php:99]: How is this function different from CountDownloads?

File: src/Repository/LogActionItemRepository.php [hunk 2]
```diff
@@ -90,4 +90,51 @@ public function countDownloads(\DateTime $start = null, \DateTime $stop = null):
 
         return $downloadCount;
     }
+
+    /**
+     * Array downloads, optional date range and also filter aware.
```
Comment (mickel1138) [inline src/Repository/LogActionItemRepository.php:95]: Fix description?

File: src/Repository/LogActionItemRepository.php (hunk 1 above)
Comment (fightingtexasaggie) [reply src/Repository/LogActionItemRepository.php]: Moved to the report.

**Review by mickel1138 — APPROVED** [review]
LGTM

**Pull request merged**